pandas/tests/frame/methods/test_cov_corr.py

```diff
@@ -191,6 +191,23 @@ def test_corr_nullable_integer(self, nullable_column, other_column, method):
         expected = pd.DataFrame(np.ones((2, 2)), columns=["a", "b"], index=["a", "b"])
         tm.assert_frame_equal(result, expected)
 
+    def test_corr_item_cache(self):
+        # Check that corr does not lead to incorrect entries in item_cache
```

Comment:
Is there an open issue for this? If not can you add a whatsnew?

Reply:
whatsnew added